apps/server/src/shared/repo/course/course.repo.integration.spec.ts

```diff
@@ -59,6 +59,7 @@ describe('course repo', () => {
 				'description',
 				'name',
 				'school',
+				'shareToken',
```

Comment:
shareToken should a optional response and also optional in property interface or?